File: BaekjoonOnlineJudge/1978/1978.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cnt = sc.nextInt();
        int answer = 0;

        for(int i = 0; i < cnt; i++){
            int input = sc.nextInt();
            answer += isPrime(input);
        }

        System.out.println(answer);
    }

    static int isPrime(int num){
        if(num == 1) return 0;

        for(int i = 2; i < num -1; i++){
            if(num % i == 0) return 0;
        }

        return 1;
    }
}

File: BaekjoonOnlineJudge/1978/1978.kt
un main(args: Array<String>) {
    val cnt = readLine()!!.toInt()
    val arr = readLine()!!.split(" ").map { it.toInt() }
    var answer = 0;

    for (i in 0 until cnt){
        answer += isPrime(arr.get(i))
    }

    println(answer)
}

fun isPrime(num: Int): Int{
    if(num == 1) return 0

    for (i in 2 until num - 1){
        if(num % i == 0) return 0
    }

    return 1;
}
